great_expectations/expectations/core/expect_column_to_exist.py

```diff
@@ -30,33 +30,85 @@
 
 
 class ExpectColumnToExist(BatchExpectation):
-    """Expect the specified column to exist.
+    """Checks for the existence of a specified column within a table.
 
     expect_column_to_exist is a \
     [Batch Expectation](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_batch_expectations).
 
+    BatchExpectations are one of the most common types of Expectation. They are evaluated for an entire Batch, and answer a semantic question about the Batch itself.
+
     Args:
         column (str): \
             The column name.
 
     Other Parameters:
-        column_index (int or None): \
+        column_index (int or None, optional): \
             If not None, checks the order of the columns. The expectation will fail if the \
             column is not in location column_index (zero-indexed).
-        result_format (str or None): \
+        result_format (str or None, optional): \
             Which output mode to use: BOOLEAN_ONLY, BASIC, COMPLETE, or SUMMARY. \
             For more detail, see [result_format](https://docs.greatexpectations.io/docs/reference/expectations/result_format).
-        catch_exceptions (boolean or None): \
+        catch_exceptions (boolean or None, optional): \
             If True, then catch exceptions and include them as part of the result object. \
             For more detail, see [catch_exceptions](https://docs.greatexpectations.io/docs/reference/expectations/standard_arguments/#catch_exceptions).
-        meta (dict or None): \
+        meta (dict or None, optional): \
             A JSON-serializable dictionary (nesting allowed) that will be included in the output without \
```

Comment:
```suggestion
            A JSON-serializable dictionary (nesting allowed) that is included in the output without \
```
Review and rewrite to avoid _will_ where applicable. 

![image](https://github.com/great-expectations/great_expectations/assets/104205257/2ce4d981-849e-448c-af6e-7ea168b39cb2)